src/main/java/org/prebid/server/handler/AccountCacheInvalidationHandler.java

```diff
@@ -18,21 +19,27 @@
     private static final String ACCOUNT_ID_PARAM = "account";
 
     private final CachingApplicationSettings cachingApplicationSettings;
+    private final String endpoint;
 
-    public AccountCacheInvalidationHandler(CachingApplicationSettings cachingApplicationSettings) {
+    public AccountCacheInvalidationHandler(CachingApplicationSettings cachingApplicationSettings, String endpoint) {
         this.cachingApplicationSettings = Objects.requireNonNull(cachingApplicationSettings);
+        this.endpoint = Objects.requireNonNull(endpoint);
     }
 
     @Override
-    public void handle(RoutingContext context) {
-        final HttpServerRequest request = context.request();
-        final String accountId = request.getParam(ACCOUNT_ID_PARAM);
+    public void handle(RoutingContext routingContext) {
+        final String accountId = routingContext.request().getParam(ACCOUNT_ID_PARAM);
+        final Consumer<HttpServerResponse> responseConsumer;
```

Comment:
To my taste calling `HttpUtil.executeSafely` twice with consumers inlined bears less cognitive load and hence preferred.